### ISSUE TITLE
Only start the callout when one of the AssignedPlayers is in range

### ISSUE THREAD

**Issue by HB-2012** [KDani-99/FivePD-API#9]
I have noticed that callouts get started when _any_ player is in range of the callout. In my opinion it would be better if the call only starts when one of the assigned players are near.

The use case for this is that I'd like to create some logic that allows for the responding officer to see a person comitting a crime. If another officer happens to be in range of said call, but is doing something else and does not know the existance of the specific call, the officer is not going to pay attention or do anything with the call. Resulting in the original responding officer driving up to a scene that is completely finished.

Given, this is a little fix on the callout developer side, but I think it'd be neat if it was fixed on the FivePD side (perhaps even with a boolean flag in the callout).

Workaround for now:

```c#
public override void OnStart(Ped closest)
{
    if (!AssignedPlayers.Contains(closest))
        return;

    base.OnStart(closest);

    // etc
}
```

**Comment by KDani-99** [comment]
Thanks for opening an issue.
I'll have to take a look on that, because it supposed to only start with assigned players.

**Comment by KDani-99** [comment]
Found the issue, will be fixed in 1.0.4. 
Thanks.
